server/api/utils/singletons/db.py

```diff
@@ -44,3 +52,17 @@ def initialize_db(override_db=None):
         db.initialize(db_session)
     finally:
         db_session.close()
+
+
+def _mask_dsn(dsn):
```

Comment:
Maybe you should move this function to `server.api.utils.db.mysql.MySQLUtil`

Reply:
@roei3000b I included it here because it can also be SQLite, not just MySQL